djangoproject/djangoapp/views.py

```diff
@@ -1,3 +1,12 @@
 from django.shortcuts import render
+from rest_framework.decorators import api_view, permission_classes
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+from .serializers import UserSerializer
+from django.contrib.auth.models import User
 
-# Create your views here.
+@api_view()
+@permission_classes((IsAuthenticated,))
+def user(request):
+    serializer = UserSerializer(context={'request': request}, instance=request.user)
```

Comment:
ovdje ti je dovoljno samo `UserSerializer(request.user)`

Reply:
Probao sam na taj nacin i radi, tako da cu sada ispraviti to.